### ISSUE TITLE
typo in poller/interfaces.py

### ISSUE THREAD

**Issue by mechkhans** [netenglabs/suzieq#331]
**Describe the bug**
A typo on Line 118 causes the code to break:
https://github.com/netenglabs/suzieq/blob/master/suzieq/poller/services/interfaces.py#L118

**To Reproduce**

run sq-poller for eos 

> `sq-poller -k -D inventory --run-once=gather --output-dir=<>`

```
  File "/root/.local/lib/python3.7/site-packages/suzieq/poller/services/interfaces.py", line 118, in _clean_eos_data
    ip = elem["adddress"] + "/" + elem["maskLen"]
```

**Comment by anubisg1** [comment]
probably related

```
# tail -f sq-poller.log
2021-03-10 09:23:21,097 - suzieq.poller - WARNING - log level WARNING
2021-03-10 09:23:29,471 - suzieq.poller - INFO - Suzieq Started
2021-03-10 09:23:29,473 - suzieq.poller - WARNING - Starting Coalescer
2021-03-10 09:23:37,913 - suzieq.poller.services.service - ERROR - Processing data failed for service interfaces on node cleaf-36-ltc
Traceback (most recent call last):
  File "/root/.local/lib/python3.7/site-packages/suzieq/poller/services/service.py", line 653, in run
    result = self.process_data(output)
  File "/root/.local/lib/python3.7/site-packages/suzieq/poller/services/service.py", line 374, in process_data
    return self.clean_data(result, data)
  File "/root/.local/lib/python3.7/site-packages/suzieq/poller/services/service.py", line 431, in clean_data
    processed_data = dev_clean_fn(processed_data, raw_data)
  File "/root/.local/lib/python3.7/site-packages/suzieq/poller/services/interfaces.py", line 469, in _clean_nxos_data
    entry['speed'] = int(speed[2:])
ValueError: invalid literal for int() with base 10: '10G'
2021-03-10 09:23:38,285 - suzieq.poller.services.service - ERROR - Processing data failed for service interfaces on node cleaf-37-ltc
Traceback (most recent call last):
  File "/root/.local/lib/python3.7/site-packages/suzieq/poller/services/service.py", line 653, in run
    result = self.process_data(output)
  File "/root/.local/lib/python3.7/site-packages/suzieq/poller/services/service.py", line 374, in process_data
    return self.clean_data(result, data)
  File "/root/.local/lib/python3.7/site-packages/suzieq/poller/services/service.py", line 431, in clean_data
    processed_data = dev_clean_fn(processed_data, raw_data)
  File "/root/.local/lib/python3.7/site-packages/suzieq/poller/services/interfaces.py", line 469, in _clean_nxos_data
    entry['speed'] = int(speed[2:])
ValueError: invalid literal for int() with base 10: '10G'
2021-03-10 09:23:39,685 - suzieq.poller.services.service - ERROR - Processing data failed for service interfaces on node bleaf-64-ltc
Traceback (most recent call last):
  File "/root/.local/lib/python3.7/site-packages/suzieq/poller/services/service.py", line 653, in run
    result = self.process_data(output)
  File "/root/.local/lib/python3.7/site-packages/suzieq/poller/services/service.py", line 374, in process_data
    return self.clean_data(result, data)
  File "/root/.local/lib/python3.7/site-packages/suzieq/poller/services/service.py", line 431, in clean_data
    processed_data = dev_clean_fn(processed_data, raw_data)
  File "/root/.local/lib/python3.7/site-packages/suzieq/poller/services/interfaces.py", line 469, in _clean_nxos_data
    entry['speed'] = int(speed[2:])
ValueError: invalid literal for int() with base 10: '40G'
2021-03-10 09:23:39,694 - suzieq.poller.services.service - ERROR - Processing data failed for service interfaces on node bleaf-63-ltc
Traceback (most recent call last):
  File "/root/.local/lib/python3.7/site-packages/suzieq/poller/services/service.py", line 653, in run
    result = self.process_data(output)
  File "/root/.local/lib/python3.7/site-packages/suzieq/poller/services/service.py", line 374, in process_data
    return self.clean_data(result, data)
  File "/root/.local/lib/python3.7/site-packages/suzieq/poller/services/service.py", line 431, in clean_data
    processed_data = dev_clean_fn(processed_data, raw_data)
  File "/root/.local/lib/python3.7/site-packages/suzieq/poller/services/interfaces.py", line 469, in _clean_nxos_data
    entry['speed'] = int(speed[2:])
ValueError: invalid literal for int() with base 10: '40G'
2021-03-10 09:23:40,241 - suzieq.poller.services.service - ERROR - Processing data failed for service interfaces on node cleaf-34-ltc
Traceback (most recent call last):
  File "/root/.local/lib/python3.7/site-packages/suzieq/poller/services/service.py", line 653, in run
    result = self.process_data(output)
  File "/root/.local/lib/python3.7/site-packages/suzieq/poller/services/service.py", line 374, in process_data
    return self.clean_data(result, data)
  File "/root/.local/lib/python3.7/site-packages/suzieq/poller/services/service.py", line 431, in clean_data
    processed_data = dev_clean_fn(processed_data, raw_data)
  File "/root/.local/lib/python3.7/site-packages/suzieq/poller/services/interfaces.py", line 469, in _clean_nxos_data
    entry['speed'] = int(speed[2:])
ValueError: invalid literal for int() with base 10: '10G'
2021-03-10 09:23:40,255 - suzieq.poller.services.service - ERROR - Processing data failed for service interfaces on node cleaf-35-ltc
Traceback (most recent call last):
  File "/root/.local/lib/python3.7/site-packages/suzieq/poller/services/service.py", line 653, in run
    result = self.process_data(output)
  File "/root/.local/lib/python3.7/site-packages/suzieq/poller/services/service.py", line 374, in process_data
    return self.clean_data(result, data)
  File "/root/.local/lib/python3.7/site-packages/suzieq/poller/services/service.py", line 431, in clean_data
    processed_data = dev_clean_fn(processed_data, raw_data)
  File "/root/.local/lib/python3.7/site-packages/suzieq/poller/services/interfaces.py", line 469, in _clean_nxos_data
    entry['speed'] = int(speed[2:])
ValueError: invalid literal for int() with base 10: '10G'
2021-03-10 09:23:40,269 - suzieq.poller.services.service - ERROR - Processing data failed for service interfaces on node cleaf-33-ltc
Traceback (most recent call last):
  File "/root/.local/lib/python3.7/site-packages/suzieq/poller/services/service.py", line 653, in run
    result = self.process_data(output)
  File "/root/.local/lib/python3.7/site-packages/suzieq/poller/services/service.py", line 374, in process_data
    return self.clean_data(result, data)
  File "/root/.local/lib/python3.7/site-packages/suzieq/poller/services/service.py", line 431, in clean_data
    processed_data = dev_clean_fn(processed_data, raw_data)
  File "/root/.local/lib/python3.7/site-packages/suzieq/poller/services/interfaces.py", line 469, in _clean_nxos_data
    entry['speed'] = int(speed[2:])
ValueError: invalid literal for int() with base 10: '10G'
2021-03-10 09:23:40,485 - suzieq.poller.services.service - ERROR - Processing data failed for service interfaces on node cleaf-02-ltc
Traceback (most recent call last):
  File "/root/.local/lib/python3.7/site-packages/suzieq/poller/services/service.py", line 653, in run
    result = self.process_data(output)
  File "/root/.local/lib/python3.7/site-packages/suzieq/poller/services/service.py", line 374, in process_data
    return self.clean_data(result, data)
  File "/root/.local/lib/python3.7/site-packages/suzieq/poller/services/service.py", line 431, in clean_data
    processed_data = dev_clean_fn(processed_data, raw_data)
  File "/root/.local/lib/python3.7/site-packages/suzieq/poller/services/interfaces.py", line 469, in _clean_nxos_data
    entry['speed'] = int(speed[2:])
ValueError: invalid literal for int() with base 10: '40G'
2021-03-10 09:23:40,505 - suzieq.poller.services.service - ERROR - Processing data failed for service interfaces on node cleaf-06-ltc
Traceback (most recent call last):
  File "/root/.local/lib/python3.7/site-packages/suzieq/poller/services/service.py", line 653, in run
    result = self.process_data(output)
  File "/root/.local/lib/python3.7/site-packages/suzieq/poller/services/service.py", line 374, in process_data
    return self.clean_data(result, data)
  File "/root/.local/lib/python3.7/site-packages/suzieq/poller/services/service.py", line 431, in clean_data
    processed_data = dev_clean_fn(processed_data, raw_data)
  File "/root/.local/lib/python3.7/site-packages/suzieq/poller/services/interfaces.py", line 469, in _clean_nxos_data
    entry['speed'] = int(speed[2:])
ValueError: invalid literal for int() with base 10: '40G'
2021-03-10 09:23:41,144 - suzieq.poller.services.service - ERROR - Processing data failed for service interfaces on node cleaf-04-ltc
Traceback (most recent call last):
  File "/root/.local/lib/python3.7/site-packages/suzieq/poller/services/service.py", line 653, in run
    result = self.process_data(output)
  File "/root/.local/lib/python3.7/site-packages/suzieq/poller/services/service.py", line 374, in process_data
    return self.clean_data(result, data)
  File "/root/.local/lib/python3.7/site-packages/suzieq/poller/services/service.py", line 431, in clean_data
    processed_data = dev_clean_fn(processed_data, raw_data)





10:24
  File "/root/.local/lib/python3.7/site-packages/suzieq/poller/services/interfaces.py", line 469, in _clean_nxos_data
    entry['speed'] = int(speed[2:])
ValueError: invalid literal for int() with base 10: '40G'
2021-03-10 09:23:41,169 - suzieq.poller.services.service - ERROR - Processing data failed for service interfaces on node cleaf-01-ltc
Traceback (most recent call last):
  File "/root/.local/lib/python3.7/site-packages/suzieq/poller/services/service.py", line 653, in run
    result = self.process_data(output)
  File "/root/.local/lib/python3.7/site-packages/suzieq/poller/services/service.py", line 374, in process_data
    return self.clean_data(result, data)
  File "/root/.local/lib/python3.7/site-packages/suzieq/poller/services/service.py", line 431, in clean_data
    processed_data = dev_clean_fn(processed_data, raw_data)
  File "/root/.local/lib/python3.7/site-packages/suzieq/poller/services/interfaces.py", line 469, in _clean_nxos_data
    entry['speed'] = int(speed[2:])
ValueError: invalid literal for int() with base 10: '40G'
2021-03-10 09:23:41,191 - suzieq.poller.services.service - ERROR - Processing data failed for service interfaces on node cleaf-05-ltc
Traceback (most recent call last):
  File "/root/.local/lib/python3.7/site-packages/suzieq/poller/services/service.py", line 653, in run
    result = self.process_data(output)
  File "/root/.local/lib/python3.7/site-packages/suzieq/poller/services/service.py", line 374, in process_data
    return self.clean_data(result, data)
  File "/root/.local/lib/python3.7/site-packages/suzieq/poller/services/service.py", line 431, in clean_data
    processed_data = dev_clean_fn(processed_data, raw_data)
  File "/root/.local/lib/python3.7/site-packages/suzieq/poller/services/interfaces.py", line 469, in _clean_nxos_data
    entry['speed'] = int(speed[2:])
ValueError: invalid literal for int() with base 10: '40G'
2021-03-10 09:23:41,993 - suzieq.poller.services.service - ERROR - Processing data failed for service interfaces on node cleaf-03-ltc
Traceback (most recent call last):
  File "/root/.local/lib/python3.7/site-packages/suzieq/poller/services/service.py", line 653, in run
    result = self.process_data(output)
  File "/root/.local/lib/python3.7/site-packages/suzieq/poller/services/service.py", line 374, in process_data
    return self.clean_data(result, data)
  File "/root/.local/lib/python3.7/site-packages/suzieq/poller/services/service.py", line 431, in clean_data
    processed_data = dev_clean_fn(processed_data, raw_data)
  File "/root/.local/lib/python3.7/site-packages/suzieq/poller/services/interfaces.py", line 469, in _clean_nxos_data
    entry['speed'] = int(speed[2:])
ValueError: invalid literal for int() with base 10: '40G'
2021-03-10 09:23:42,674 - suzieq.poller.services.service - ERROR - Processing data failed for service interfaces on node cleaf-38-ltc
Traceback (most recent call last):
  File "/root/.local/lib/python3.7/site-packages/suzieq/poller/services/service.py", line 653, in run
    result = self.process_data(output)
  File "/root/.local/lib/python3.7/site-packages/suzieq/poller/services/service.py", line 374, in process_data
    return self.clean_data(result, data)
  File "/root/.local/lib/python3.7/site-packages/suzieq/poller/services/service.py", line 431, in clean_data
    processed_data = dev_clean_fn(processed_data, raw_data)
  File "/root/.local/lib/python3.7/site-packages/suzieq/poller/services/interfaces.py", line 469, in _clean_nxos_data
    entry['speed'] = int(speed[2:])
ValueError: invalid literal for int() with base 10: '10G'
2021-03-10 09:23:52,814 - suzieq.poller.services.service - ERROR - Processing data failed for service mlag on node spine-02-ltc
Traceback (most recent call last):
  File "/root/.local/lib/python3.7/site-packages/suzieq/poller/services/service.py", line 653, in run
    result = self.process_data(output)
  File "/root/.local/lib/python3.7/site-packages/suzieq/poller/services/service.py", line 374, in process_data
    return self.clean_data(result, data)
  File "/root/.local/lib/python3.7/site-packages/suzieq/poller/services/service.py", line 431, in clean_data
    processed_data = dev_clean_fn(processed_data, raw_data)
  File "/root/.local/lib/python3.7/site-packages/suzieq/poller/services/mlag.py", line 85, in _clean_nxos_data
    entry['peerLink'] = expand_nxos_ifname(entry['peerLink'])
  File "/root/.local/lib/python3.7/site-packages/suzieq/utils.py", line 622, in expand_nxos_ifname
    if ifname.startswith('Eth') and 'Ether' not in ifname:
AttributeError: 'NoneType' object has no attribute 'startswith'
2021-03-10 09:23:52,986 - suzieq.poller.services.service - ERROR - Processing data failed for service mlag on node spine-01-ltc
Traceback (most recent call last):
  File "/root/.local/lib/python3.7/site-packages/suzieq/poller/services/service.py", line 653, in run
    result = self.process_data(output)
  File "/root/.local/lib/python3.7/site-packages/suzieq/poller/services/service.py", line 374, in process_data
    return self.clean_data(result, data)
  File "/root/.local/lib/python3.7/site-packages/suzieq/poller/services/service.py", line 431, in clean_data
    processed_data = dev_clean_fn(processed_data, raw_data)
  File "/root/.local/lib/python3.7/site-packages/suzieq/poller/services/mlag.py", line 85, in _clean_nxos_data
    entry['peerLink'] = expand_nxos_ifname(entry['peerLink'])
  File "/root/.local/lib/python3.7/site-packages/suzieq/utils.py", line 622, in expand_nxos_ifname
    if ifname.startswith('Eth') and 'Ether' not in ifname:
AttributeError: 'NoneType' object has no attribute 'startswith'
```

**Comment by ddutt** [comment]
Fixed with commit 9219c2334ab3fbb8eb5f6e4631d554277f67974d, c321638d6d619b2f685e6e9539ca086cd36efac1
